capstone/src/main/java/com/google/univiz/CollegeData.java

```diff
@@ -0,0 +1,86 @@
+package com.google.univiz;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * CollegeData is a class representing a single university/college/institution.
+ */
+@AutoValue
+abstract class CollegeData {
+
+  abstract int id();
+
+  abstract String name();
+
+  abstract String city();
+
+  /**
+   * Returns a boolean that tells whether the institution is the main campus or not, true or false.
+   */
+  abstract boolean isMainCampus();
+
+  /**
+   * Returns an enum UrbanizationDegree that describes the institution's degree of urbanzation, or
+   * how populated the surrounding areas are around the institution, from large city to rural.
+   */
+  abstract UrbanizationDegree urbanizationDegree();
+
+  abstract float latitude();
+
+  abstract float longitude();
+
+  /**
+   * Returns an enum CarnegieSizeDegree that represents the institution's carnegie size degree, or
+   * the highest degree certificate available at that institution, from not applicable to
+   * exclusively graduate/professional.
+   */
+  abstract CarnegieSizeDegree carnegieSizeDegree();
+
+  abstract float avgSat();
+
+  abstract int numOfUndergrads();
+
+  abstract int avgCost();
+
+  abstract float ratioOfMen();
+
+  abstract float ratioOfWomen();
+
+  static Builder builder() {
+    return new AutoValue_CollegeData.Builder().setUrbanizationDegree(0);
```

Comment:
This line may need to change to 
```suggestion
    return new AutoValue_CollegeData.Builder().setUrbanizationDegree(UrbanizationDegree.NOT_AVAILABLE);
```
or something similar since likely now the integer 0 doesn't directly translate to an `UrbanizationDegree` anymore. But even if this code is still valid, the above suggestion makes use of the name of the type, and may be more readable.